core/src/main/scala/org/http4s/syntax/package.scala

```diff
@@ -18,6 +18,7 @@ package org.http4s
 
 package object syntax {
   object all extends AllSyntaxBinCompat
+  @deprecated("import is no longer needed", "0.22.3")
```

Comment:
What about the `translate` methods?  I can understand how `orNotFound` works (and it's exciting that it does!).

Reply:
I don't think it should be any different? They're all on types that are aliases for `Kleisli` with a `Response` parameter. I'll make sure there's test coverage though.

Reply:
Hmm, I think you're right, if it winds its way through those type aliases.  :+1: to making sure it's tested.